File: WebApplication/src/modules/application/application-one/models/application-one-search-result.dto.ts
import { CommitState } from "../../common/enums/commit-state.enum";
import { ApplicantType } from "../enums/applicant-type.enum";
import { ApplicationOneType } from "../enums/application-one-type.enum";

export class ApplicationOneSearchResultDto {
  id: number;
  rootId: number;
  registerNumber: string;
  applicationOneType: ApplicationOneType;
  applicantType: ApplicantType;
  commitState: CommitState;
  applicantName: string;
  basicName: string;
  createDate: Date;
  hasApplicationTwo: boolean;
  applicationTwoCommitState: CommitState;
}

File: Zopoesht.Application/Applications/ApplicationsOne/Dtos/ApplicationOneSearchResultDto.cs
using Zopoesht.Application.Applications.Common.Dtos;
using Zopoesht.Data.Applications.ApplicationOne.Enums;
using Zopoesht.Data.Common.Enums;

namespace Zopoesht.Application.Applications.ApplicationsOne.Dtos
{
    public class ApplicationOneSearchResultDto
    {
        public int Id { get; set; }

        public int RootId { get; set; }

        public string RegisterNumber { get; set; }

        public IEnumerable<ApplicationAuthorityDto> ApplicationAuthorities { get; set; }

        public ApplicationOneType ApplicationOneType { get; set; }

        public ApplicantType ApplicantType { get; set; }

        public CommitState CommitState { get; set; }

        public string ApplicantName { get; set; }

        public string BasicName { get; set; }

        public DateTime CreateDate { get; set; }

        public bool HasApplicationTwo { get; set; }

        public CommitState? ApplicationTwoCommitState { get; set; }
    }
}
